diff --git a/node_modules/svelte-preprocess-cssmodules/dist/processors/parseStyle.js b/node_modules/svelte-preprocess-cssmodules/dist/processors/parseStyle.js
index b0e0135..34ca2bf 100644
--- a/node_modules/svelte-preprocess-cssmodules/dist/processors/parseStyle.js
+++ b/node_modules/svelte-preprocess-cssmodules/dist/processors/parseStyle.js
@@ -11,7 +11,7 @@ const parseStyle = (content, filename, cssModuleList) => {
             const generatedClass = match.replace(patterns_1.PATTERN_CLASSNAME(className), () => `.${cssModuleList[className]}`);
             return generatedClass.indexOf(':global(') !== -1
                 ? generatedClass
-                : `:global(${generatedClass})`;
+                : `:global(${generatedClass})`.replace(',', '') + (generatedClass.includes(',') ? ',' : '')
         });
     });
     return {
